bin/update-ips.bash

```diff
@@ -51,6 +51,61 @@ diffIPs() {
     return ${DIFF_RETURN}
 }
 
+# Get IPs in given field and config file
+getConfigIPS() {
+    local source_field="$1"
+    local config_file="$2"
+
+    local configIPS=()
+
+    for ip in $(yq4 e "$source_field | .[]" "$config_file"); do
+        if [[ $ip != */32 ]]; then
+            configIPS+=("${ip}")
+        fi
+    done
+
+    echo "${configIPS[@]}"
+}
+
+# Insert processes IPs
+updateConfigFile() {
+    local source_field="$1"
+    local config_file="$2"
+    local ip="$3"
+
+    if [[ $ip =~ "/" ]]; then
+        yq4 -i "${source_field}"' |= . + ["'"${ip}"'"]' "${config_file}"
+    else
+        yq4 -i "${source_field}"' |= . + ["'"${ip}"'/32"]' "${config_file}"
+    fi
+}
+
+# Check if IPs fits into subnets
+performIPCheck() {
+    local kubectlIP="$1"
+    local configIP="$2"
+
+    # Try to see if ip belongs to subnet - if not, exit(1)
+    if python3 -c "import ipaddress; exit(0) if ipaddress.ip_address('${kubectlIP}') in ipaddress.ip_network('${configIP}') else exit(1)"
```

Comment:
What happens if the `ipaddress.ip_address('${kubectlIP}')` part gets a range bigger than /32? Will it return only if it's a strict subset or if there's any match?

Reply:
${kubectlIP} needs to be a valid IP(as in no extras). If not:

"ValueError: '172.16.0.43/32' does not appear to be an IPv4 or IPv6 address" for example.